main.tf

```diff
@@ -231,6 +231,21 @@ resource "google_compute_address" "static_nginx_ip" {
   }
 }
 
+resource "helm_release" "nginx_controller" {
+   # see https://kubernetes.github.io/ingress-nginx/deploy/#using-helm
```

Comment:
think there is an additional space before the comment

Reply:
Ah, yep that sneaky space got in there. 😈 Removing it.